Pividori Lab, Cole Lab, JRaviLab, Krishnan Lab, Way Lab, Gignoux Lab (and ui improvements)

vincerubinetti opened the pull request:
@miltondp @bytesizesci @jananiravi @arjunkrish @gwaybio Please take a look at the diff text and images, and the Netlify preview link below.

- add lab logos in pr title
- increase description limit
- replace svg filter with more performant css filter
- simplify css selectors to help safari out with bugs and performance
- improve tooltip sizing
- improve dna helix draw-in

Submitter checklist:

- [x] My changes follow the guidelines in the `README.md`.
- [x] My changes appear correctly in the preview link (if applicable).

Reviewer checklist:

- [ ] The changes follow the guidelines in the `README.md`.
- [ ] The changes appear correctly in the preview link (if applicable).

vincerubinetti committed:
Pividori, Cole, JRavi, Krshnan, Way, Gignoux Lab

gwaybio commented on groups.json:
```diff
@@ -1,12 +1,42 @@
+  },
+  {
+    "name": "Way Lab",
+    "description": "Biomedical data science and software for translational research",
```
```suggestion
    "description": "Biomedical data science and software for drug discovery and translational research",
```

vincerubinetti replied:
The text here needs to be less than about 80 characters because it needs to fit into a tooltip. I'll be truncated if it's longer than that. Also the text version is for printing.

gwaybio reviewed (approved):
LGTM - although i don't see the "text" version of the logo in netlify

miltondp commented on groups.json:
```diff
@@ -1,12 +1,42 @@
+  },
+  {
+    "name": "Pividori Lab",
+    "description": "Novel machine-learning for precision medicine",
```
```suggestion
    "description": "Computational methods for a systems biology approach to precision medicine",
```

miltondp replied:
74 characters now :)

miltondp reviewed (approved):
Looks great, @vincerubinetti!

vincerubinetti committed:
increase description limit

vincerubinetti committed:
improve ui

vincerubinetti commented:
Okay I've reworked how the tooltips behave so we can fit more text in there, and I've removed the hard character limit. But still, please keep your descriptions short. They _will_ be cut off after 3 lines of text.

jananiravi commented on groups.json:
```diff
@@ -1,12 +1,42 @@
+    "link": "https://www.thekrishnanlab.org"
+  },
+  {
+    "name": "J Ravi Lab",
```
minor: JRavi could be together (also filename)

jananiravi commented on groups.json:
```diff
@@ -1,12 +1,42 @@
+  },
+  {
+    "name": "J Ravi Lab",
+    "description": "Computational study of microbial genotypes, phenotypes, and diseases",
```
study of --> methods to study

miltondp commented on groups.json:
```diff
@@ -1,12 +1,42 @@
+  },
+  {
+    "name": "Pividori Lab",
+    "description": "Innovative computational methods for a systems biology approach to precision medicine",
```
```suggestion
    "description": "Computational methods for a systems biology approach to precision medicine",
```

miltondp reviewed (approved):
I shortened the description of my lab.

jananiravi reviewed (approved):
looks great -- thanks for adding us!
added minor edits.

vincerubinetti committed:
jravilab tweaks

vincerubinetti committed:
pividorilab tweak

Co-authored-by: Milton Pividori <miltondp@gmail.com>

vincerubinetti committed:
fix cole lab

vincerubinetti committed:
fix krishnan lab

vincerubinetti committed:
remove exif meta

vincerubinetti committed:
restore gignoux lab
